cpp/recognition/planar/HopcroftTarjan.cpp

```diff
@@ -0,0 +1,358 @@
+/*
+ * HopcroftTarjan.cpp
+ *
+ *  Created on: 24.03.2024
+ *      Author: Dzianis Lahunou
+ *      Ported by: Krzysztof Turowski (krzysztof.szymon.turowski@gmail.com)
+ */
+
+#include <iostream>
+#include <list>
+#include <set>
+#include <networkit/linkprediction/NeighborhoodUtility.hpp>
+#include <recognition/planar/PlanarGraphRecognition.hpp>
+
+namespace Koala {
+
+    class Block {
+    private:
+        std::list<int> Latt, Ratt;
+        std::list<NetworKit::WeightedEdge> Lseg, Rseg;
```

Comment:
Suggestion: please use full names instead of "att" and "seg", it'll be easier to read your code to outsiders.